internal/scripts/scripts.go

```diff
@@ -208,3 +197,23 @@ func (c *ContentValue) List(thread *starlark.Thread, fn *starlark.Builtin, args
 	}
 	return starlark.NewList(values), nil
 }
+
+func (c *ContentValue) Symlink(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (Value, error) {
+	var slTarget, slLinkPath starlark.String
+	err := starlark.UnpackArgs("Content.symlink", args, kwargs, "target", &slTarget, "linkpath", &slLinkPath)
+	if err != nil {
+		return nil, err
+	}
+
+	target := slTarget.GoString()
+	linkPath := slLinkPath.GoString()
+	linkPath, err = c.RealPath(linkPath, nil)
+	if err != nil {
+		return nil, c.polishError(slLinkPath, err)
+	}
+	err = os.Symlink(target, linkPath)
```

Comment:
Let's please force the target to be relative and inside the container too.

Reply:
This refers to outdated code. Currently, target is passed through `c.RealPath` but the result is ignored, only error is checked. So whether target is absolute or relative, it's still resolved to a path inside container root. I'm not sure what you meant exactly by forcing taget to be relative. Do you suggest mangling the link target line `ln -r` does?